E2E/assertions/assertions.ts

```diff
@@ -0,0 +1,39 @@
+const { I, game } = inject();
+
+export = {
+  async checkIfTheLinkWasCopied (currentUrl) {
```

Comment:
Parameter 'currentUrl' implicitly has an 'any' type. As you are using it as string, change it to 'currentUrl: string'